### ISSUE TITLE
NVDA does not work with Greek. It is just reading out the letters of each word.

### ISSUE THREAD

**Issue by Christosz93** [nvaccess/nvda#13882]
### Steps to reproduce:
Enable Greek in NVDA.
### Actual behavior:
It reads out every letter, merely spelling the word, instead of reading it. 
F.i. if we have the word Ακρόπολη. it reads it as: ALPHA KAPA RO OMIKRON PI OMIKRON LAMDA HTA. 
### Expected behavior:
It should read "Acropoli" instead of all the letters that compose the word.
#### NVDA installed/portable/running from source:
Installed from https://www.nvaccess.org/download/
#### NVDA version:
2022.1
#### Windows version:
Window 7 pro.
#### Name and version of other software in use when reproducing the issue:
All.
#### Other information about your system:
Laptop, 8GB RAM, 64-bit.
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by CyrilleB79** [comment]
Hi @Christosz93

Which synthesizer are you using? And with which voice?

To have the text read in Greek, you have to use a Greek synthesizer.

I have testedwith eSpeak Greek and Greek (Ancient) voices. Both read the word "Ακρόπολη" as a word and they do not spell it.

**Comment by CyrilleB79** [comment]
Closing since I cannot reproduce and have no feedback.

@Christosz93, if you still have an issue, please comment here with appropriate requested information and we can reopen.

**Comment by Adriani90** [comment]
Yeah this is totally dependend on what synthesizer people are using. eSpeak can read it as a word, but i.e. Onecore doesn't recognize the letters at all.

I am thinking to add greek letters to symbols.dic with propper pronounciation because they have propper unicode implementation and are separated from greek letters used in math or science , but don't know when i will have time to do it.
And this will depend on the implementation of the feature to report unicode names on demand like by pressing nvda+dot four times or nvda+comma two times.

Here is a code example that would translate the unicode names into the windows locale, but the question remains on how to make NVDA generate these translation requests when retrieving the unicode names with the key stroke.
https://stackoverflow.com/questions/47656304/are-there-standardized-translations-of-unicode-character-names

However, even having the unicode names in english on demand would be a big step forward, the CLDR data base has already many localized character names so we could retrieve the information from there at least for a significant variety of unicode characters.
